### DISCUSSION TITLE
Output IP Addresses to Text File

### DISCUSSION THREAD

**Discussion by keysplayer** [DigitalRuby/IPBan#295] [Ideas]
Hello - can IPBan be configured to output only the failed IP addresses to a text file? Many hardware firewalls can call a threat feed/text file of IP addresses into a deny policy. As an example, I'd like to minimize VPN brute force attempts on my firewall by blocking repeat failed addresses. SSLVPN is configured on my firewall to authenticate via RADIUS/NPS with Accounting and I can see the public IP addresses in Event Viewer. IPBan is displaying all output in the logfile and I just need the public addresses so the firewall can import the list and deny access to the VPN.

**Comment by jjxtra** [comment]
Process to run on ban in the config can do this

**Comment by keysplayer** [comment]
Thanks for the quick response. What process do you recommend that will read the address and export?
